requirements.txt

```diff
@@ -23,7 +21,6 @@ docutils==0.15.2
 ecdsa==0.14.1
 Faker==5.8.0
```

Comment:
Is this dependency needed here, or could it be moved to `requirements-test.txt`?

Reply:
@jsf9k Yes, at the moment it could be moved to requirements-test.txt, but we actually had a new request that we'll be utilizing this library in our actual app, so I'm moving it here for now.

Reply:
OK, got it.